documentation/api/current/search.js

```diff
@@ -51,13 +51,15 @@ function getURLPrefix(ui) {
         } else if ((ui.item.category === catTypes && ui.item.p) || ui.item.category === catMembers) {
             $.each(packageSearchIndex, function(index, item) {
                 if (ui.item.p == item.l) {
-                    urlPrefix = item.m + slash;
```

Comment:
Minor but we try not to make whitespace changes if you're not modifying the line

Reply:
@sudersan01 Thanks for updating the commits! As @Ashpan mentioned, it would be good to remove the white space changes on this line.